### DISCUSSION TITLE
KAIST CS431: Concurrent Programming - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#515] [Announcements]
# KAIST CS431: Concurrent Programming - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%BC%96%E7%A8%8B%E5%85%A5%E9%97%A8/cs431/

**Comment by TephrocactusMYC** [comment]
在B站上有一个翻译版本，在这门课被加入CSDIY之前我一直看的B站版本。
【001-【KAIST CS431/CS492】【Rust并发编程】【Introduction】】 https://www.bilibili.com/video/BV1Xp4y1w7xG/?share_source=copy_web&vd_source=89fdd6c33222facf5d82736180e256cc

**Comment by featherwit001** [comment]
看到这个标题，我就想起来之前刚好在B站看的课程，前段时间刚刚更完

**Comment by ghost** [comment]
感谢

**Comment by LWX-Research** [comment]
CS70的课，没账号怎么办

**Comment by clffffffff** [comment]
CS61A的recording需要有账号，佬们怎么解决的？

**Reply by rinevard** [reply]
在油管搜John DeNero，再点开播放列表试试。

**Reply by kylehuang23** [reply]
感谢！

**Comment by Rimor4** [comment]
请问这份cs自学指南适用于软工吗（大一新生求问）

**Reply by dslvbaie** [reply]
非常适合 计科和软工几乎没有区别

**Reply by Rimor4** [reply]
好的感谢回复

**Comment by nick199910** [comment]
有没有小伙伴一起做homework的呀，一起加个微信讨论嘞，15734904757

**Comment by ZhitongGuo** [comment]
请问大家，是否有agile development相关的课？大家是如何学习敏捷开发流程的？谢谢！

**Comment by WeiliHe** [comment]
楼主您你好，我看到你也有cs186的课程资料，但是Proj那块他罪行Update的已经不能pull skeleton code了，请问你这边有他6个projects 的starter code 吗

**Comment by dengyunsheng250** [comment]
这个有一起做的人吗？

**Reply by cathzzr2** [reply]
有！

**Reply by 520Enterprise** [reply]
你学什么课，咱可以交流下，我邮箱是 enty520.study@outlook.com

**Comment by AdAstraAbyssoque** [comment]
**感谢**

**Comment by thinkerhui** [comment]
感谢分享，受益匪浅。
              --from SYSU,SSE

**Comment by XuhaoQin02** [comment]
请问是否可以加入robotics的课程呢

**Comment by learnerlittle** [comment]
作为一个工作了3年被开掉已经失业半年的游戏客户端傻狗，最近致力于探索怎么样才能在计算机行业干的更久一点，看到了您的总结，让我又看到了一点未来的曙光，非常感谢

**Comment by DanielDaniel2201** [comment]
刚刚开始上这门课，欢迎一起交流~ qq: 1667938478
